scipy/stats/BiasedUrn.pxd

```diff
@@ -0,0 +1,31 @@
+cdef extern from "biasedurn/fnchyppr.cpp":
```

Comment:
These made the code work, but is it the right way to do it?

Reply:
I would add them instead as sources in setup.py.

Reply:
Ok, I will try removing them from here and adding them into the list of sources of the extension.

Reply:
Thanks! That looks a lot cleaner.